Diagnostics/FreeBusyChecker/Functions/htmlContent.ps1

```diff
@@ -0,0 +1,1029 @@
+﻿# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+[Diagnostics.CodeAnalysis.SuppressMessageAttribute('PSUseDeclaredVarsMoreThanAssignments', '', Justification = 'Variables are being used in functions')]

```

Comment:
This isn't needed.

Reply:
Removed from code. Correction will be available when changes are pushed.